src/utils/api/transactions.js

```diff
@@ -25,14 +27,15 @@ export const getTransactions = ({
   return activePeer.transactions.get(params);
 };
 
-export const getSingleTransaction = ({ activePeer, id }) => requestToActivePeer(activePeer, 'transactions/get', { id });
+export const getSingleTransaction = ({ activePeer, id }) => activePeer.transactions.get({ id });
 
-export const unconfirmedTransactions = (activePeer, address, limit = 20, offset = 0, orderBy = 'timestamp:desc') =>
-  requestToActivePeer(activePeer, 'transactions/unconfirmed', {
-    senderId: address,
-    recipientId: address,
-    limit,
-    offset,
-    orderBy,
+export const unconfirmedTransactions = (activePeer, address, limit = 20, offset = 0, sort = 'timestamp:desc') =>
+  new Promise(() => {
```

Comment:
Please don't wrap a promise as the getTransactions is already returning a Promise.